packages/py-api/py_api/environment.py

```diff
@@ -8,5 +8,6 @@
 MONGO_URI = getenv("MONGOURI", "")
 
 IS_OFFLINE = eval_bool(getenv("IS_OFFLINE", False))
+IS_PROD = eval_bool(getenv("IS_PROD", False))
```

Comment:
Is this because we are going to have a dev environment as well? I mean  IS_OFFLINE env var is Flase, aren't we in PROD?

Reply:
Correct. `IS_OFFLINE` won't particularly mean that we're on production.